features/fixtures/mazerunner/jvm-scenarios/src/main/java/com/bugsnag/android/mazerunner/scenarios/FeatureFlagScenario.kt

```diff
@@ -0,0 +1,31 @@
+package com.bugsnag.android.mazerunner.scenarios
+
+import android.content.Context
+import com.bugsnag.android.Bugsnag
+import com.bugsnag.android.Configuration
+
+class FeatureFlagScenario(
```

Comment:
This scenario doesn't cover adding multiple flags or clearing them - shouldn't it cover the whole API surface (hopefully within the one scenario)?